### DISCUSSION TITLE
Spaceship: Create keys? (e.g. Sign in with Apple)

### DISCUSSION THREAD

**Discussion by tupakapoor** [fastlane/fastlane#19274] [Q&A]
Hello!

Does Spaceship (or some other action) support listing/creating keys in the Keys section of the Developer Portal? For reference I'm referring to the section located here: https://developer.apple.com/account/resources/authkeys/list

Thanks!

**Comment by lucgrabowski** [comment]
You could use `Spaceship::Portal::Key` to list and create keys. https://github.com/fastlane/fastlane/blob/master/spaceship/lib/spaceship/portal/key.rb
Handling Sign in with Apple key is not ready yet, there is an open pull request with it though - https://github.com/fastlane/fastlane/pull/17803
